### ISSUE TITLE
freqz does not support FIRFilters

### ISSUE THREAD

**Issue by ethomag** [JuliaDSP/DSP.jl#185]
```
julia> f = FIRFilter([.25, 0.5, 0.25]);
julia> freqz(f)
ERROR: MethodError: no method matching freqz(::DSP.Filters.FIRFilter{DSP.Filters.FIRStandard{Float64}})

julia> Pkg.status()
[ ... ]
 - DSP                           0.3.4
```
I am running Julia 0.6

**Comment by martinholters** [comment]
Workaround would be `freqz(PolynomialRatio(f.h, [1.0]))`, but a `freqz` method that takes a `FIRFilter` (or maybe a `FIRFilter{<:FIRStandard}`) directly surely wouldn't hurt.

**Comment by ethomag** [comment]
Thanks Martin!  What if I have a decimating FIR filter ?
````
julia> f = FIRFilter([.25, 0.5, 0.25], 1//2);
````

**Comment by martinholters** [comment]
Yea, in general, if `ratio ≠ 1`, it is unclear whether `freqz` should compute the frequency response viewed from the input or output side of the filter. That's why I included the possible restriction to `FIRFilter{<:FIRStandard}` above :wink:

**Comment by ethomag** [comment]
Good point! In MATLAB, I believe `freqz` delivers a result viewed from the output side of the filter.  I guess that would conform to the "rule of least surprise".
